packages/renderer/src/lib/kube/KubernetesDashboardResourceCard.svelte

```diff
@@ -12,7 +12,8 @@ interface Props {
 
 let { type, Icon, activeCount, count, link }: Props = $props();
 
-function openLink(): void {
+async function openLink(): Promise<void> {
+  await window.telemetryTrack('kubernetes.dashboard.resource', { type: type });
```

Comment:
question: link is not sent ?

Reply:
I figure we were more interested in which button was clicked than where it goes, but yeah, zero harm in adding in. Done via new commit.